source/Digitisers/src/DDPlanarDigiProcessor.cc

```diff
@@ -216,14 +217,12 @@ void DDPlanarDigiProcessor::processEvent( LCEvent * evt ) {
     
     CellIDEncoder<TrackerHitPlaneImpl> cellid_encoder( lcio::LCTrackerCellID::encoding_string() , trkhitVec ) ;
 
-    LCCollectionVec* relCol = new LCCollectionVec(LCIO::LCRELATION);
-    // to store the weights
-    LCFlagImpl lcFlag(0) ;
-    lcFlag.setBit( LCIO::LCREL_WEIGHTED ) ;
-    relCol->setFlag( lcFlag.getFlag()  ) ;
-    
+    // Relation collection TrackerHit, SimTrackerHit
+    LCCollection* thsthcol  = 0;
+    UTIL::LCRelationNavigator thitNav = UTIL::LCRelationNavigator( LCIO::TRACKERHIT, LCIO::SIMTRACKERHIT );
```

Comment:
Should this be `LCIO::TRACKERHITPLANE` type? (@gaede what do you think?)
Especially tricky with the EDM4hep conversion and lack of inheritance there.